app/src/main/java/de/timbolender/fefereader/ui/SearchableActivity.kt

```diff
@@ -0,0 +1,63 @@
+package de.timbolender.fefereader.ui
+
+import android.app.SearchManager
+import android.content.Intent
+import android.os.Build
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.ViewModelProvider
+import androidx.paging.PagedList
+import de.timbolender.fefereader.db.DataRepository
+import de.timbolender.fefereader.db.Post
+import de.timbolender.fefereader.network.Updater
+import de.timbolender.fefereader.viewmodel.SearchViewModel
+import java.io.IOException
+import java.text.ParseException
+import java.util.concurrent.CompletableFuture
+import java.util.concurrent.Executor
+import java.util.concurrent.Executors
+
+class SearchableActivity : PostListActivity() {
```

Comment:
The activity has no back arrow in the bar. I would like to keep it consistent with the DetailsActivity.